### PR TITLE
[bitnami/wordpress] Improved WordPress and MariaDB READMEs

### PR DESCRIPTION
**Pull request by vikram-bitnami**
**Description of the change**

Condensed information and linked to chart documentation on docs.bitnami.com

**Benefits**

Shorter, easier-to-read file
Focus on key information with links to related resources

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb/README.md [hunk 1]
```diff
@@ -288,68 +288,37 @@ To modify the MariaDB version used in this chart you can specify a [valid image
...
-    ports:
-      - name: portname
-        containerPort: 1234
-```
+Similarly, additional containers can be added to MariaDB pods using the `initContainers` parameter. [See an example of configuring and using init containers](https://docs.bitnami.com/kubernetes/infrastructure/mariadb/administration/configure-use-init-containers/).
 
 ## Persistence
 
-The [Bitnami MariaDB](https://github.com/bitnami/bitnami-docker-mariadb) image stores the MariaDB data and configurations at the `/bitnami/mariadb` path of the container.
+The [Bitnami MariaDB](https://github.com/bitnami/bitnami-docker-mariadb) image stores the MariaDB data and configurations at the `/bitnami/mariadb` path of the container.
```
Comment (marcosbc) [inline bitnami/mariadb/README.md:309]: We should remove the trailing newline:
```suggestion
The [Bitnami MariaDB](https://github.com/bitnami/bitnami-docker-mariadb) image stores the MariaDB data and configurations at the `/bitnami/mariadb` path of the container.
```

File: bitnami/wordpress/README.md [hunk 2]
```diff
@@ -298,7 +298,7 @@ kubectl exec $(kubectl get pods -l app.kubernetes.io/name=wordpress -o jsonpath=
 kubectl exec $(kubectl get pods -l app.kubernetes.io/name=wordpress -o jsonpath='{.items[2].metadata.name}') -c wordpress -- wp maintenance-mode activate
 ```
 
-### Adding extra environment variables
+### Additional environment variables
```
Comment (marcosbc) [inline bitnami/wordpress/README.md:301]: The example mentions `kong`, could you fix that?

File: bitnami/wordpress/README.md [hunk 3]
```diff
@@ -313,32 +313,11 @@ Alternatively, you can use a ConfigMap or a Secret with the environment variable
...
-  - name: extraPort
-    port: 11311
-    targetPort: 11311
-```
-
-### Using an external database
+### External database support
 
-Sometimes you may want to have Wordpress connect to an external database rather than installing one inside your cluster, e.g. to use a managed database service, or use run a single database server for all your applications. To do this, the chart allows you to specify credentials for an external database under the [`externalDatabase` parameter](#parameters). You should also disable the MariaDB installation with the `mariadb.enabled` option. For example with the following parameters:
+Sometimes, you may want to have WordPress connect to an external database rather than installing one inside your cluster, e.g. to use a managed database service, or to use a single database server for all your applications. To do this, the chart allows you to specify credentials for an external database with the [`externalDatabase` parameter](#parameters). You should also disable the MariaDB installation with the `mariadb.enabled` option. Here is an example:
```
Comment (marcosbc) [inline bitnami/wordpress/README.md:320]: I see too many commas in the first sentence (`,`), making it hard to follow. Could you rephrase it so it is easier to follow?

**Review by marcosbc — COMMENTED** [review]
Small requests, apart from those it looks good to me!

**Review by juan131 — APPROVED** [review]
LGTM
